src/modules/lists.test.js

```diff
@@ -39,21 +42,53 @@ describe('Test', () => {
     expect(localStorage.getItem('tasks')).toHaveLength(1);
   });
 
+  test('Add a new item to todo list', () => {
+    addTaskMock('Item two');
+    expect(toDoListArray).toHaveLength(2);
+    expect(localStorage.getItem('tasks')).toHaveLength(2);
+  });
+
   test('Add a new item to DOM todo list', () => {
     renderToDoListMock(toDoListArray, ul);
     const lis = Array.from(ul.children);
-    expect(lis).toHaveLength(1);
+    expect(lis).toHaveLength(2);
+  });
+
+  test('Edit task description', () => {
+    const newValue = 'updated task description';
+    editTaskMock(0, newValue);
+    expect(toDoListArray[0].description).toEqual(newValue);
+  });
+
+  test('Mark a todo as completed', () => {
+    markTaskAsCompletedMock(1);
+    expect(toDoListArray[1].completed).toBeTruthy();
+    expect(toDoListArray[1].completed).toEqual(true);
+    expect(localStorage.getItem('tasks')[1].completed).toBeTruthy();
+    expect(localStorage.getItem('tasks')[1].completed).toEqual(true);
   });
 
   test('Remove only one item', () => {
     deleteTaskMock(0);
-    expect(toDoListArray).toHaveLength(0);
-    expect(localStorage.getItem('tasks')).toHaveLength(0);
+    expect(toDoListArray).toHaveLength(1);
+    expect(localStorage.getItem('tasks')).toHaveLength(1);
```

Comment:
- [ ] On your readme file the MIT link is broken, kindly fix it 
![image](https://user-images.githubusercontent.com/62705947/216363878-922d6b61-c326-40fd-9b88-6cb29d263497.png)